src/xdp_iphash_to_cpu_user.c

```diff
@@ -532,6 +543,9 @@ int main(int argc, char **argv)
 		case 'a':
 			add_all_cpus = true;
 			break;
+		case 's':
+			qsize = strtoul(optarg, NULL, 0);
```

Comment:
Check the return value, maybe? :)

Reply:
Well, I would have to check `errno` for invalid or out-of-range input values.
I was thinking, the kernel will reject this if it is out-of-range.
Looking at manual, I can see that it can return 0 on `EINVAL`, which will be bad actually.

Reply:
Notice on kernel side qsize==0 is same as deleting cpumap entry, which is why is it extra bad